godtools/App/Features/ToolDetails/Presentation/ToolDetails/ToolDetailsViewModel.swift

```diff
@@ -289,17 +289,8 @@ extension ToolDetailsViewModel {
     }
     
     func segmentTapped(index: Int) {
-        selectedSegment = segmentTypes[index]
         
-        // NOTE: This is a temporary fix that solves an issue when switching back to the about section, a user is unable to scroll through all of the text in the
-        // about section. This is basically acting as a way to force the view to refresh again after 0.1 seconds.
-        // This has something to do with using UIViewRepresentable that wraps a UITextView.  If we were using a SwiftUI Text element this issue wouldn't exist.
-        // Once minimum iOS 15 is supported see if TextWithLinks can be dropped.  See GT-1670. ~Levi
-        if selectedSegment == .about {
-            DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
```

Comment:
So adding `layoutIfNeeded` ended up solving this problem too?

Reply:
Hey @rachaelblue it did thankfully!  The UITextView is using an intrinsic content sizing so just needed to force the layout to update itself again.